examples/ecommerce/pages/index.tsx

```diff
@@ -231,13 +218,12 @@ const Home: NextPage = () => {
     const suggestionsKey = 'q.suggestions';
     const getSuggestions = (values: any) => (values && values[suggestionsKey] ? values[suggestionsKey].split(',') : []);
 
-    pipeline.current.autocomplete
-      .search(request.serialize(), session.next(request.serialize()))
-      .then(([, values]: any[]) => {
-        setState({
-          suggestions: getSuggestions(values).slice(0, 10),
-        });
+    pipeline.current.autocomplete.search(request.serialize()).then(([response, values]: any[]) => {
+      searchIOAnalytics.updateQueryId(response.queryId);
```

Comment:
Not sure if we should be setting the query ID based on responses from the autocomplete pipeline but I'd have to ask @JasonBerry to be sure one way or the other.

Reply:
I think that this is okay, as if a redirect occurs as a result of autocomplete then this query will be the one bound to the redirect, but if we're just taken to another search result page then the query used on that search result page will be the next one stored and will be bound should any of the results be clicked on.

Reply:
So that means this is good to go? Merging in the next couple of hours if no one objects.